app/services/pii/attributes.rb

```diff
@@ -23,7 +23,7 @@ def self.new_from_hash(hash)
       attrs = new
       hash.with_indifferent_access.
         slice(*members).
-        each { |key, val| attrs[key] = val }
+        each { |key, val| attrs[key] = val.is_a?(String) ? val&.squish : val }
```

Comment:
Is the nil-safe operator necessary if we've already checked that it's a string and therefore not nil?